lib/src/SlashRemover.cc

```diff
@@ -96,33 +98,13 @@ void SlashRemover::initAndStart(const Json::Value& config)
                         removeExcessiveSlashes(newPath);
                         break;
                 }
+                req->setPath(newPath);
```

Comment:
I believe this is intended for the forwarder, shouldn't it be in the else statement? If we are redirecting, we don't need to alter the request